hvpy/io.py

```diff
@@ -0,0 +1,25 @@
+from enum import Enum
+from pydantic import BaseModel
+
+BASE_URL = "https://api.helioviewer.org/v2/"
```

Comment:
In future, we will need this to be overridable/settable by a user. Maybe even as a OS environment. We can leave it here for this PR.